### PR TITLE
Decorate remaining undecorated tests with `testmode`

### PR DESCRIPTION
**Pull request by hannes-ucsc**

### CODE REVIEW

**Comment by ttung** [comment]
I think some classes were just decorated entirely. Do the method-level ones need to be there still?

**Comment by hannes-ucsc** [comment]
If in doubt, we should stick with method-level decorations. Even if all methods in class are decorated, that could be incidental. If a test class is inherently either integration or standalone, I made a class level decoration.

**Comment by hannes-ucsc** [comment]
I captured the concerns in #820 and would like to go ahead with this as is.

**Comment by ttung** [comment]
I clarified my viewpoint in #820, but I'll stamp this PR and let you decide how you want to do it. :)

File: tests/test_smoketest.py [hunk 1]
```diff
@@ -42,7 +43,7 @@ def run(command, runner=check_call, **kwargs):
         parser.exit(RED(f'{parser.prog}: Exit status {e.returncode} while running "{command}". Stopping.'))
 
 
-@infra.testmode.integration
+@testmode.standalone
```
Comment (Bento007) [inline tests/test_smoketest.py]: shouldn't this be integration

Comment (ttung) [reply tests/test_smoketest.py]: great catch. :)

Comment (hannes-ucsc) [reply tests/test_smoketest.py]: Oops. Fixed.

**Pull request merged**